### ISSUE TITLE
PHP Deprecation

### ISSUE THREAD

**Issue by eparovyshnaya** [eclipse-passage/passage-website#1]
> Denis Roy:
```
Your project is still using PHP on your website (not the download area on [download.eclipse.org](http://download.eclipse.org/)) and that support is being removed next month.

Please see the deprecation timeline and background here[0]

Please migrate your pages to static html as soon as possible. You can also use Jenkins and Hugo [1] to publish websites based on a build mechanism, or you can substitute your website with the standard page offered by the PMI - https://projects.eclipse.org/

The EF IT team is standing by to assist. Please  file a HelpDesk [2] if you need assistance or support.

-> PHP is also deprecated on [download.eclipse.org](http://download.eclipse.org/), but you have until the end of 2025 to make those changes.
```

In scope of this ticket any agreed actions are to be taken to unhook [.dev/passage](https://eclipse.dev/passage/) from using PHP. 

Static page looks reasonable choise.

**Comment by eparovyshnaya** [comment]
#### Resources worth mentioning:

|  Resource                       |  Link                                                                       |
| ------------------------- | ------------------------------------------------------ |
| Passage Documentation | https://eclipse-passage.github.io/passage-docs/ |
| Current state | https://projects.eclipse.org/projects/technology.passage | 
| Passage Mailing List | https://www.eclipse.org/lists/passage-dev |
| Created and maintained by ArSysOp | https://arsysop.ru |

#### General description 
can be limited to simple
> The Eclipse Passage™ project aims to provide rich and easily adaptable capabilities to declare and control licensing constraints.

**Comment by eparovyshnaya** [comment]
solved with #2

**Comment by eparovyshnaya** [comment]
@ruspl-afed 
The site does not use PHP anymore.
Please take further actions if needed.
